motionplan/tpSpaceRRT.go

```diff
@@ -1054,8 +1054,8 @@ func extractTPspacePath(startMap, goalMap map[node]node, pair *nodePair) []node
 			q: []referenceframe.Input{
 				goalReached.Q()[0],
 				goalReached.Q()[1],
+				goalReached.Q()[3],
```

Comment:
Why are these out of order?

Reply:
The tpspace algorithm calculates the goal tree in reverse. This is where those inputs are rectified to be with regard to the forwards tree.